lib/loga/railtie.rb

```diff
@@ -153,26 +153,38 @@ def self.call
 
       def call
         ensure_subscriptions_attached
+        subscribe_to_action_mailer
         remove_log_subscriptions
       end
 
       private
 
+      def subscribe_to_action_mailer
+        LogSubscribers::ActionMailer.attach_to :action_mailer
+      end
+
       # Ensure LogSubscribers are attached when available
       def ensure_subscriptions_attached
         ActionView::Base       if defined?(ActionView::Base)
         ActionController::Base if defined?(ActionController::Base)
+        ActionMailer::Base     if defined?(ActionMailer::Base)
       end
 
       # rubocop:disable Metrics/LineLength
       def remove_log_subscriptions
         ActiveSupport::LogSubscriber.log_subscribers.each do |subscriber|
-          case subscriber
-          when defined?(ActionView::LogSubscriber) && ActionView::LogSubscriber
-            unsubscribe(:action_view, subscriber)
-          when defined?(ActionController::LogSubscriber) && ActionController::LogSubscriber
-            unsubscribe(:action_controller, subscriber)
-          end
+          defined_subscriber = ->(log_subscriber) { defined?(log_subscriber) && log_subscriber }
+
+          entity = case subscriber
+                   when defined_subscriber.call(ActionView::LogSubscriber)
```

Comment:
This is not the same. `defined?` is a keyword https://docs.ruby-lang.org/en/2.5.0/syntax/miscellaneous_rdoc.html#label-defined-3F

You cannot do this because the argument will be evaluated and it may start raising errors like `NameError: uninitialized constant ActionView`